api/src/main/java/org/pmiops/workbench/google/DirectoryServiceImpl.java

```diff
@@ -157,21 +160,23 @@ private String randomString(){
   }
 
   protected void sendPasswordEmail(String contactEmail, String password, User user) {
-    WorkbenchConfig workbenchConfig = configProvider.get();
-    Properties props = new Properties();
-    Session session = Session.getDefaultInstance(props, null);
+    MandrillMessage message = new MandrillMessage();
+    RecipientAddress toAddress = new RecipientAddress();
+    toAddress.setEmail(contactEmail);
+    message.setTo(Arrays.asList(toAddress));
     String messageBody = "Your new account is: " + user.getPrimaryEmail() +
       "\nThe password for your new account is: " + password;
+    message.setHtml(messageBody);
+    message.setSubject("Your new All of Us Account");
+    // using this address as a placeholder right now.
+    // do not think it is right since I'm getting a
+    // unsigned rejection right now, but the api should work
+    // once we have the domains/from Email straightened out
+    message.setFromEmail("donotreply@allofus.org");
     try {
-      Message msg = new MimeMessage(session);
-      msg.setFrom(new InternetAddress(workbenchConfig.admin.verifiedSendingAddress));
-      msg.addRecipient(Message.RecipientType.TO, new InternetAddress(
-        contactEmail, user.getName().getFullName()));
-      msg.setSubject("Your new All of Us Account");
-      msg.setText(messageBody);
-      mailServiceProvider.get().send(msg);
-    } catch (MessagingException | UnsupportedEncodingException e) {
-      throw new EmailException("Error sending password email", e);
+      mandrillService.sendEmail(message);
```

Comment:
The bulk of this method should be abstracted to the mail service implementation class. I also think it would be best to abstract the from email address to the configuration instead of hard-coding it here. That way they can be changed at any time just by updating the config in the credentials GCS bucket.

Reply:
So should I store the email in the GCS bucket? Or in our configs?
As mentioned in the comments I actually just did this as a place holder, and I wasn't sure if we'd want different emails from different environments (helpful to devs?)
Also - I actually thought of this whole method as a bit of a placeholder to be better implemented when we actually plug in the REAL intro email (note this is still just the quick dirty email I just swapped out the google mail service for mandrill....

Reply:
We 100% need a `sendPasswordEmail` method. It should look a lot like what it does here, but instead be in the mail sending class instead of this class.

Reply:
I'm suggesting that we store the value of `message.setFromEmail("donotreply@allofus.org")` in the configuration instead of hard coding it here. That way, it is no longer a placeholder and is a real thing.